### ISSUE TITLE
F# Query Expressions example: System.Data.Linq does not exist for net5.0

### ISSUE THREAD

**Issue by kurt-mueller-osumc** [dotnet/docs#23278]
The example provided utilizes these namespaces:

```F#
open System
open Microsoft.FSharp.Data.TypeProviders
open System.Data.Linq.SqlClient
open System.Linq
```

Unfortunately, `System.Data.Linq` does not exist on `net5.0`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c3514796-1d23-6213-51c8-a90e399a5afd
* Version Independent ID: 518ea6eb-b99d-deac-eca3-4a2be6b4baa9
* Content: [Query Expressions - F#](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/query-expressions)
* Content Source: [docs/fsharp/language-reference/query-expressions.md](https://github.com/dotnet/docs/blob/main/docs/fsharp/language-reference/query-expressions.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

**Comment by cartermp** [comment]
Yeah, ugh. This whole document likely needs a rewrite here. I don't think the examples are very good to show now. It won't be terribly easy though, since it would require contriving up some new samples. Maybe something with `#r "nuget:..."` on SqlProvider?

**Comment by BillWagner** [comment]
Fixed in #48751
